pkg/nmi/server/server.go

```diff
@@ -513,24 +491,6 @@ func copyHeader(dst, src http.Header) {
 	}
 }
 
-func handleTermination() {
```

Comment:
Whats is the windows equivalent of this?

Reply:
Yes. I am adding the code to handle pod delete event.

Reply:
My question is, should we need to, since I believe endpoint would be deleted after pod delete. Let me ask this to the folks in the thread.

Reply:
My question is, should we need to, since I believe endpoint would be deleted after pod delete. Let me ask this to the folks in the thread.

Reply:
Yeah. There are two cases here: 
1. When pod is deleted, it will fire a pod deletion event. If the endpoint would be deleted after pod deleted, then it will be easy for us to do nothing, here, right? Otherwise, our code needs to call DeleteRoute through proxy agent. It is a little tricky here since once the pod is deleted, we may not be able to get the podip in pod delete event, which mean we may need to cache this info during pod creation event. During pod deletion, we can always get the pod name, which will help us to filter the corresponding pod ip if we cache both info during pod creation event.

2. When the nmi pod is deleted or stopped for some reason, should we list all the existing pods and call DeleteRoute through proxy agent.

Reply:
For now, I added the change for no 2.

Reply:
both 1 and 2 are added.